### PR TITLE
Adds the Banana Phone (you can now message the Clown Planet!)

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

Adds the Bananaphone. It allows the clown to message the Clown Planet. It is located in the theatre in every map and looks like this.

Use it only for emergencies.

Credit for @Partheo for supplying me with a sprite

#### Changelog

:cl:  
rscadd: Adds the Banana Phone. The Clown can message the Clown Planet now. 
imageadd: Adds Partheo's Banana Phones sprite
/:cl:

### CODE REVIEW

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -371,7 +371,30 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	to_chat(usr, "<span class='danger'>Message sent. Pray you made the right choice.</span>")
 	usr.log_talk(input, LOG_SAY, tag="Syndicate announcement")
 	deadchat_broadcast(" has messaged the Syndicate using the red phone, \"[input]\" at <span class='name'>[get_area_name(usr, TRUE)]</span>.", "<span class='name'>[usr.real_name]</span>", usr)
-
+	
+/obj/item/phone/banana
+	name = "banana phone"
+	desc = "Emergency telecommunications line to the Clown Planet. Use for emergencies only!"
```
Comment (Partheo) [inline code/game/objects/items/weaponry.dm]: Redundant "emergency". Maybe "**Direct** telecommunications line to the Clown Planet. Use for emergencies only!" instead?

**Review by Partheo — APPROVED** [review]

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (swissloaf) [reply code/game/objects/items/weaponry.dm]: good idea

**Comment by Partheo** [comment]
it hurts my soul that phones are in weaponry.dm

**Comment by ynot01** [comment]
> it hurts my soul that phones are in weaponry.dm

its because before phones were made functional they were just low damage weapons that made funny sound

**Review by patpol4 — CHANGES_REQUESTED** [review]
Your changes revert cryopods, please update your branch.

**Review by patpol4 — APPROVED** [review]
Enjoy your vacation Swiss

**Pull request merged**